seqr/views/apis/dashboard_api_tests.py

```diff
@@ -28,9 +31,8 @@
 ]
 
 
-class DashboardPageTest(AuthenticationTestCase):
-    fixtures = ['users', '1kg_project']
-
+class TestHelpers(object):
```

Comment:
I strongly prefer the previous approach of having a top level `class DashboardPageTest(object):`. Why would you change this? I think its good for this test helper class to be unable to run without being mixed with an authentication case, so I don't like the idea of it being able to run with `BasicAuthTestCase`

Reply:
If the class `DashboardPageTest` doesn't inherit any class like `BasicAuthTestCase` then many attributes used with the `DashboardPageTest` class are undefined. It is okay for running but it is hard to write or update the tests because the IDE can't provide any hints for the attributes.

Reply:
This nested class syntax is really hard to read. I disagree that the IDE hints make this easier to update, I think the weird nesting makes it harder, and makes it really unclear if the tests are supposed to be able to run on their own or not. I strongly prefer we change it back.

Reply:
Here are some suggestions for how to trick the IDE into knowing what class something is meant to be, if you feel really strongly about it:
https://intellij-support.jetbrains.com/hc/en-us/community/posts/205818219-How-to-tell-PyCharm-that-a-class-is-a-mixin-specifically-for-class-X

Reply:
I will try `assert isinstance(self, BaseAuthTestCase)`.

Reply:
okay I will wait to review until these changes are in

Reply:
I will just change them back and use `assert isinstance(self, AuthenticationTestCase)` when needs.